Std/Data/BitVec/Lemmas.lean

```diff
@@ -117,6 +117,23 @@ private theorem lt_two_pow_of_le {x m n : Nat} (lt : x < 2 ^ m) (le : m ≤ n) :
   else
     simp [h]
 
+
+/-! ### msb -/
+
+theorem msb_decide (x : BitVec (Nat.succ w)) : BitVec.msb x = decide (x.toNat ≥ 2^w) := by
```

Comment:
```suggestion
theorem msb_eq_decide (x : BitVec (Nat.succ w)) : BitVec.msb x = decide (2 ^ w ≤ x.toNat) := by
```
Even though this is not a simp-lemma, it makes sense to use the simp-normal form (which prefers `<=` over `>=`). Also, the name should have `eq`